ui.apps/src/main/content/jcr_root/apps/core/fd/components/form/wizard/v1/wizard/clientlibs/editorhook/.content.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+      jcr:primaryType="cq:ClientLibraryFolder"
+      allowProxy="{Boolean}true"
+      categories="[core.wcm.components.wizard.editor.hook.v1.hook,cq.authoring.editor.hook]"
+      dependencies="[cq.authoring.editor.core]"/>
```

Comment:
its dependent on `window.CQ.CoreComponents.panelcontainer.v1.registry` also. pls include corresponding clientlib in deps as well